ocs_ci/utility/rosa.py

```diff
@@ -262,3 +294,9 @@ def delete_oidc_provider(cluster_id):
     """
     cmd = f"rosa delete oidc-provider -c {cluster_id}"
     utils.run_cmd(cmd)
+
+
+if __name__ == "__main__":
```

Comment:
Should this be part of the rosa.py module?

Reply:
removed